src/secret.rs

```diff
@@ -91,7 +91,7 @@ pub struct DisplaySecret {
 impl fmt::Debug for DisplaySecret {
```

Comment:
Unrelated but I think this should be called just `Display` - same as `std::path::Display`

Reply:
I'm going to leave this as is for now, can re-visit if you feel strongly.